tests/backends/tests.py

```diff
@@ -745,6 +745,19 @@ def test_cursor_contextmanager_closing(self):
             self.assertIsInstance(cursor, CursorWrapper)
         self.assertTrue(cursor.closed)
 
+    @skipUnlessDBFeature('test_db_allows_multiple_connections')
+    def test_is_usable_before_after_connection(self):
+        """
+        Test that is_usable() doesn't crash when the connection is not open yet.
+        Test that is_usable() doesn't crash when the connection is already closed.
+        """
+        # without a connection, the database is unusable
+        self.assertFalse(connection.is_usable())
+        with connection.cursor():
+            self.assertTrue(connection.is_usable())
```

Comment:
Also, if you are using a context manager, it seems like you want to assert calling `is_usable()` right before you close (so after the context manager closes).